src/background.ts

```diff
@@ -82,22 +84,24 @@ async function generateQuery(message: { context: string, url: string, dims: [num
       "dimensions": message.dims,
       "context": message.context,
       "language": "en",
-      "capabilities": [],
+      "capabilities": capabilities,
       "renderers": renderers
     } as IMAGERequest;
   });
 }
 
 async function generateLocalQuery(message: { context: string, dims: [number, number], image: string }): Promise<IMAGERequest> {
   let renderers = await getRenderers();
+  const debugMode = await isDebugModeEnabled();
+  const capabilities = debugMode ? [CAPABILITIES.debugMode] : [];
```

Comment:
Same as above, may be right to refactor renderers and capabilities out into a helper function here? Not blocking on this either.

Reply:
Thanks for the suggestion. I have done the changes.